commands/client-pause.md

```diff
@@ -18,7 +18,7 @@ For the `WRITE` mode, some commands have special behavior:
 * `PFCOUNT`: Will block client.
 * `WAIT`: Acknowledgments will be delayed, so this command will appear blocked.
 
-This command is useful as it makes able to switch clients from a Redis instance to another one in a controlled way. For example during an instance upgrade the system administrator could do the following:
+This command is useful as it makes able to switch clients from a Valkey instance to another one in a controlled way. For example during an instance upgrade the system administrator could do the following:
```

Comment:
In this doc there is a reference to Redis 6.2, we can drop that as it's not relevant since all versions of Valkey have client pause write. For that reason, you should also update the example to be `CLIENT PAUSE WRITE`.